src/DynamoRevit/ViewModel/RevitWatch3DViewModel.cs

```diff
@@ -221,7 +223,10 @@ private IEnumerable<GeometryObject> Tessellate(Curve curve)
             {
                 var xyz0 = new XYZ(verts[i] * conv, verts[i + 1] * conv, verts[i + 2] * conv);
                 var xyz1 = new XYZ(verts[i + 3] * conv, verts[i + 4] * conv, verts[i + 5] * conv);
-
+                if (xyz0.DistanceTo(xyz1) < DocumentManager.Instance.CurrentUIApplication.Application.ShortCurveTolerance)
+                {
+                    continue;
+                }
```

Comment:
If a curve of 1' in length is tessellated into 1000 segments, then the curve won't be visible.

Shall we check the length of the curve, if it is too small then it won't even be faceted? If we do facet it, shall we use Polyline.Create() to create a polyline based on the facted points?